build.gradle.kts

```diff
@@ -24,6 +25,7 @@ repositories {
 // Dependencies are managed with Gradle version catalog - read more: https://docs.gradle.org/current/userguide/platforms.html#sub:version-catalog
 dependencies {
 //    implementation(libs.annotations)
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.5.1")
```

Comment:
I’m a bit confused why is this dependency needed - didn’t we agree to use GSON library that is already available?